tests/src/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -372,6 +372,61 @@ class WskBasicCliUsageTests
             wsk.parseJsonString(stdout).fields("annotations") shouldBe getEscapedJSONTestArgOutput
     }
 
+    behavior of "Wsk activations"
```

Comment:
since these tests are mostly the same but are changing the sub command, you can "generate" them with loop. See `GoCLINginxTests` for an example (line 86+).